### PR TITLE
Send the resource filename in the request header

### PR DESCRIPTION
**Pull request by n0izn0iz**
Currently, if you try to download a resource center file you get a file named 'data'.
With this, the file has the right name.

### CODE REVIEW

**Comment by cedricmessiant** [comment]
@vincentfretin is it ok for you ?

**Comment by vincentfretin** [comment]
I wrote a comment requesting changes.

File: assembl/views/api2/attachments.py [hunk 1]
```diff
@@ -43,7 +44,15 @@ def get_file(request):
...
     f = File.get(document.id)
-    return Response(body=f.data, content_type=str(f.mime_type))
+    quoted_filename = url_quote(f.title.encode('utf-8'))
+    return Response(
+        body=f.data,
+        content_type=str(f.mime_type),
+        content_disposition=(
+            'attachment; filename="%s"; filename*=utf-8\'\'%s' # RFC 6266
+            % (f.title, quoted_filename)
+        ).encode('utf-8')
```
[the remark this answers is not in the document]
Comment (n0izn0iz) [reply assembl/views/api2/attachments.py]: I I remove the external one I get
```
 _authdebug_view
    return view(context, request)
  File "/Users/bluenove/assembl/venv/lib/python2.7/site-packages/pyramid/viewderivers.py", line 302, in _secured_view
    return view(context, request)
  File "/Users/bluenove/assembl/venv/lib/python2.7/site-packages/pyramid/viewderivers.py", line 413, in viewresult_to_response
    result = view(context, request)
  File "/Users/bluenove/assembl/venv/lib/python2.7/site-packages/pyramid/viewderivers.py", line 147, in _requestonly_view
    response = view(request)
  File "/Users/bluenove/assembl/assembl/views/api2/attachments.py", line 53, in get_file
    % (f.title, quoted_filename)
  File "/Users/bluenove/assembl/venv/lib/python2.7/site-packages/webob/response.py", line 153, in __init__
    setattr(self, name, value)
  File "/Users/bluenove/assembl/venv/lib/python2.7/site-packages/webob/descriptors.py", line 145, in fset
    value = value.encode('latin-1')
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 22-23: ordinal not in range(256)
```

[the remark this answers is not in the document]
Comment (n0izn0iz) [reply assembl/views/api2/attachments.py]: I don't understand you second comment

[the remark this answers is not in the document]
Comment (n0izn0iz) [reply assembl/views/api2/attachments.py]: oh, got it
```
return Response(
        body=f.data,
        content_type=str(f.mime_type),
        content_disposition=
            'attachment; filename="%s"; filename*=utf-8\'\'%s' # RFC 6266
            % (f.title.encode('utf-8'), quoted_filename)
```
?

[the remark this answers is not in the document]
Comment (n0izn0iz) [reply assembl/views/api2/attachments.py]: or maybe f.title.encode('ascii', 'replace')

[the remark this answers is not in the document]
Comment (vincentfretin) [reply assembl/views/api2/attachments.py]: you can keep utf-8 that's fine

[the remark this answers is not in the document]
Comment (n0izn0iz) [reply assembl/views/api2/attachments.py]: from https://tools.ietf.org/html/rfc6266#section-4.3
>The parameters "filename" and "filename*" differ only in that
>"filename*" uses the encoding defined in [RFC5987], allowing the use
>of characters not present in the ISO-8859-1 character set
>([ISO-8859-1]).
 So I think it's mandatory, I ended up using .encode('ISO-8859-1', 'replace')

**Comment by cedricmessiant** [comment]
is it ready to merge?

**Comment by vincentfretin** [comment]
The changes looks ok, but why did you change attachmend by inline now? The difference if I remember is that for pdf if inline it will open it in the browser instead of downloading it. This is what you want?

**Comment by n0izn0iz** [comment]
woops, test artifact

**Comment by cedricmessiant** [comment]
merge !

**Pull request merged**